packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -288,9 +299,18 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
                   )}
                 </>
               )}
-              <span className="text-xs font-normal text-gray-500">
-                {tokenListInfo}
-              </span>
+              {!isL2OnlyToken && (
+                <span className="flex gap-1 text-xs font-normal text-gray-500">
+                  {tokenListInfo}
+                </span>
+              )}
+              {isL2OnlyToken && (
+                <span className="flex gap-1 text-xs font-normal text-error">
+                  {`Token contract not found on ${getNetworkName(
```

Comment:
![image](https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/ac0aa9a6-d6f9-4c29-93fe-fd4048a4ce12)

Let's also fix the overflow issue